torchao/experimental/ops/tests/test_linear_8bit_act_xbit_weight.cpp

```diff
@@ -12,14 +12,19 @@
 #include <torchao/experimental/ops/memory.h>
```

Comment:
This file runs in buck in internal CI.  Check that kleidi is covered there too (unless kleidi is not pulled in internally).

Reply:
It should be guarded with `TORCHAO_ENABLE_KLEIDI` but after kleidi lands internally we can enable it for aarch64 CI.